kyuubi-server/src/main/resources/org/apache/kyuubi/ui/static/kyuubi.js

```diff
@@ -43,5 +43,4 @@ $(function(){
 		});
 	}
 	$('.item.server.version.right.padded').val(kyuubiVersion)
-	alert(kyuubiVersion);
```

Comment:
It's better to replace this with `console.log(kyuubiVersion)`.

Reply:
it's a common function and will be invoked in many places..